components/ws-manager-bridge/src/preparing-update-emulator.ts

```diff
@@ -1,96 +0,0 @@
-/**
- * Copyright (c) 2020 Gitpod GmbH. All rights reserved.
- * Licensed under the GNU Affero General Public License (AGPL).
- * See License.AGPL.txt in the project root for license information.
- */
-import { WorkspaceDB } from "@gitpod/gitpod-db/lib/workspace-db";
-import { Disposable, DisposableCollection, WorkspaceInstance } from "@gitpod/gitpod-protocol";
-import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
-import { repeat } from "@gitpod/gitpod-protocol/lib/util/repeat";
-import { TraceContext } from "@gitpod/gitpod-protocol/lib/util/tracing";
-import { inject, injectable } from "inversify";
-import { Configuration } from "./config";
-import { MessageBusIntegration } from "./messagebus-integration";
-import { GarbageCollectedCache } from "@gitpod/gitpod-protocol/lib/util/garbage-collected-cache";
-import * as crypto from "crypto";
-
-export const PreparingUpdateEmulatorFactory = Symbol("PreparingUpdateEmulatorFactory");
-
-interface CacheEntry {
-    instance: WorkspaceInstance;
-    userId: string;
-    hash: string;
-}
-
-/**
- * The purpose of this class is to emulate WorkspaceInstance updates for workspaces instances that are not governed by this bridge.
- * It does so by polling the DB for the specific region, and if anything changed, push that update into the local messagebus.
- * This is a work-around to enable decoupling cross-cluster messagebus instances from each other.
- */
-@injectable()
-export class PreparingUpdateEmulator implements Disposable {
```

Comment:
This is entirely removed, because the whole premise of this emulator is based on multiple app clusters, which we don't have anymore.